### PR TITLE
[ticket/15529] Color groups in ACP

### PR DESCRIPTION
**Pull request by senky**
PHPBB3-15529

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15529

### CODE REVIEW

File: phpBB/includes/acp/acp_groups.php [hunk 1]
```diff
@@ -985,6 +985,7 @@ function main($id, $mode)
 					'S_GROUP_SPECIAL'	=> ($row['group_type'] == GROUP_SPECIAL) ? true : false,
 
 					'GROUP_NAME'	=> $group_name,
+					'GROUP_COLOR'	=> $row['group_colour'],
```
Comment (Crizz0) [inline phpBB/includes/acp/acp_groups.php]: We use colour instead of color.

File: phpBB/adm/style/acp_groups.html [hunk 2]
```diff
@@ -316,7 +316,7 @@ <h1>{L_SPECIAL_GROUPS}</h1>
 	<tbody>
 		<!-- ELSE -->
 		<tr>
-			<td><strong>{groups.GROUP_NAME}</strong></td>
+			<td><strong<!-- IF groups.GROUP_COLOR --> style="color: #{groups.GROUP_COLOR}"<!-- ENDIF -->>{groups.GROUP_NAME}</strong></td>
```
Comment (Crizz0) [inline phpBB/adm/style/acp_groups.html]: Color vs. Colour

**Comment by hanakin** [comment]
@Crizz0 Not for long and that’s actually a bad practice.

**Comment by Crizz0** [comment]
Why? I thought we use British English, so colour is correct, even in the same line two different ways as you can see in the changed code lines.

**Comment by hanakin** [comment]
In the copy yes but British English should not be used in code as all code is based on us English so it’s considered bad practice to mix and match ie style=“color: group-colour” is just plain confusing

File: phpBB/includes/acp/acp_groups.php (hunk 1 above)
Comment (senky) [reply phpBB/includes/acp/acp_groups.php]: Yeah, taken into account, but as far as I know, when you use this variable to assign CSS' `color` prop, then it should be `COLOR`. But if you really insist, I will rename it.

**Comment by vinny** [comment]
@hanakin `colour` is the term used in the core. Take a look for yourself at files like `includes/acp/acp_groups.php`, `memberlist.php` and `viewtopic.php`

**Comment by Crizz0** [comment]
Yes, I see the problem in the template. On the other hand, we have in the php-files this line:
`'GROUP_COLOR'	=> $row['group_colour'],`
And this looks quite strange as well.

**Comment by senky** [comment]
So what do you want me to do? Assign 2 template variables?
```
'S_GROUP_COLOUR'	=> !empty($row['group_colour']),
'COLOR'			=> $row['group_colour'],
```

**Comment by Crizz0** [comment]
No. Just keep it. I think we need a clean-up here in future, to use consistently only "COLOR" in the code.

**Pull request merged**